tests/helpers.py

```diff
@@ -88,14 +89,20 @@ def wait_for_resource_state(resource, state):
     return True
 
 
-def create_pod(interface_type=None, pvc_name=None, desired_status=constants.STATUS_RUNNING, wait=True):
+def create_pod(
```

Comment:
This is risky.
Ensure that all calls to create_pod are modified as well to fit the new signiture